wire-library/wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/SchemaTest.kt

```diff
@@ -2433,4 +2433,65 @@ class SchemaTest {
       """.trimMargin())
     }
   }
+
+  @Test fun ambiguousEnumConstants() {
+    try {
+      RepoBuilder()
+          .add("message.proto", """
+            |enum Foo {
+            |  ZERO = 0;
+            |  zero = 1;
+            |}
+          """.trimMargin())
+          .schema()
+      fail()
+    } catch (e: SchemaException) {
+      assertThat(e).hasMessage(
+          """|Ambiguous constant names (if you are using allow_alias, use the same value for these constants):
+             |  1. ZERO (/source/message.proto:2:3)
```

Comment:
The 1. and 2. bullets in these is confusing and unnecessary? Maybe use the tag values instead?

Reply:
```
Ambiguous constant names (if you are using allow_alias, use the same value for these constants):
  ZERO:0 (/source/message.proto:3:3)
  zero:1 (/source/message.proto:4:3)
  for enum Foo (/source/message.proto:1:1)
```